pkg/controller/services.go

```diff
@@ -1076,6 +1076,13 @@ func (cont *AciController) writeApicSvc(key string, service *v1.Service) {
 		p.SetAttr("nodePort", strconv.Itoa(int(port.NodePort)))
 		aobj.AddChild(p)
 	}
+	if service.ObjectMeta.Labels != nil {
```

Comment:
I'm not sure whether this is the right place but we need code somewhere that checks the APIC version and only tries to add vmmInjectedLabel as a child to APICs that support it (5.2.?+ for ns, svcs, 5.0+ for deployment, rs and pods)